lib/ical/property.js

```diff
@@ -211,7 +211,11 @@ ICAL.Property = (function() {
      * @return {Array|String}        Property value
      */
     getParameter: function(name) {
-      return this.jCal[PROP_INDEX][name];
+      if (this.jCal[PROP_INDEX][name]) {
```

Comment:
I tested it and it seemed to work (as we're not actually accessing the value of the undefined property), but |name in this.jCal[PROP_INDEX]| does seem more readable.